setup.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+from setuptools import setup
+
+requirements = [
+    'pulpcore-plugin',
+]
+
+setup(
+    name='pulp-plugin-template',
```

Comment:
one thing that could confuse a newbie is that this is a functioning setup.py. One benefit of switching to jinja would be that it would be clear that this file is a template for the plugin rather than the installer for the template.